### PR TITLE
Update font.md

### PR DESCRIPTION
**Pull request by il3ven**
Improved documentation for Adding New Symbols

### CODE REVIEW

File: en/overview/font.md [hunk 1]
```diff
@@ -128,14 +128,21 @@ To declare the font in a file, use `LV_FONT_DECLARE(my_font_name)`.
...
-2. Open the [Online font converter](https://littlevgl.com/ttf-font-to-c-array) add [FontAwesome.ttf](https://littlevgl.com/tools/FontAwesome.ttf) and add the Unicode ID of the symbol to the range field. E.g.` 0xf287` for the USB symbol.
-More symbols can be enumerated with `,`.
-3. Convert the font and copy it to your project.
-4. Convert the Unicode value to UTF8. You can do it e.g.on [this site](http://www.ltg.ed.ac.uk/~richard/utf-8.cgi?input=f287&mode=hex). For `0xf287` the *Hex UTF-8 bytes* are `EF 8A 87`.
-5. Create a `define` from the UTF8 values: `#define MY_USB_SYMBOL "\xEF\x8A\x87"`
-6. Use the symbol as the built-in symbols. `lv_label_set_text(label, MY_USB_SYMBOL)`
+The built-in symbols are created from [FontAwesome](https://fontawesome.com/) font. 
+
+1. Search symbol on [https://fontawesome.com](https://fontawesome.com). For example the [USB symbol](https://fontawesome.com/icons/usb?style=brands). Copy it's Unicode ID which is `0xf287` in this case.
+2. Open the [Online font converter](https://littlevgl.com/ttf-font-to-c-array). Add [FontAwesome.ttf](https://littlevgl.com/tools/FontAwesome.ttf).
```
Comment (kisvegabor) [inline en/overview/font.md]: Here is the up to date symbol font.
```suggestion
2. Open the [Online font converter](https://littlevgl.com/ttf-font-to-c-array). Add Add [FontAwesome.woff](https://littlevgl.com/tools/FontAwesome5-Solid+Brands+Regular.woff). . 
```

**Comment by kisvegabor** [comment]
Hi,

Thank you. I've just modified a link. Could you commit it?

**Comment by il3ven** [comment]
Sorry for the delay. I was busy.

I have made the commit. Happy to contribute. 😄

**Comment by embeddedt** [comment]
Thanks!

**Pull request merged**